webrender/src/lib.rs

```diff
@@ -95,9 +101,12 @@ extern crate core_graphics;
 #[cfg(target_os="macos")]
 extern crate core_text;
 
-#[cfg(not(target_os="macos"))]
+#[cfg(any(target_os="linux", target_os="android"))]
```

Comment:
This should be all(unix, not(target_os = "macos")) I think, since WR builds and runs on *BSD now.